stdlib/public/core/Integers.swift.gyb

```diff
@@ -2171,6 +2172,7 @@ extension FixedWidthInteger {
   
   // Homogeneous masking shift
 ${operatorComment(x.operator, False)}
+  @_semantics("optimize.sil.specialize.generic.partial.never")
```

Comment:
Why were these being specialized if they're @_transparent? Do we not inline calls to transparent functions sometimes?